### PR TITLE
Return all channel properties when using `findChannel` or `listChannel`

### PR DESCRIPTION
**Pull request by WouterWidgets**
Return all channel properties when using `findChannel` or `listChannels`.

The `priority` property is particularly useful, because it allows you to see whether the user switched the channel off (`priority` with value `0`).

### CODE REVIEW

File: src/android/FirebaseMessagingPlugin.java [hunk 1]
```diff
@@ -220,7 +220,13 @@ private void findChannel(String channelId, CallbackContext callbackContext) thro
             callbackContext.success(new JSONObject()
```
Comment (chemerisuk) [inline src/android/FirebaseMessagingPlugin.java:220]: It'd be nice to extract new static method
```java
private static JSONObject toJSON(NotificationChannel channel) throws JSONException {
```
And use it 2 times to DRY code.

File: src/android/FirebaseMessagingPlugin.java [hunk 2]
```diff
@@ -220,7 +220,13 @@ private void findChannel(String channelId, CallbackContext callbackContext) thro
...
                 .put("id", channel.getId())
                 .put("name", channel.getName())
-                .put("description", channel.getDescription()));
+                .put("description", channel.getDescription())
+                .put("importance", channel.getImportance())
+                .put("badge", channel.canShowBadge())
```
Comment (chemerisuk) [inline src/android/FirebaseMessagingPlugin.java:225]:
```suggestion
                .put("showBadge", channel.canShowBadge())
```

File: src/android/FirebaseMessagingPlugin.java [hunk 3]
```diff
@@ -220,7 +220,13 @@ private void findChannel(String channelId, CallbackContext callbackContext) thro
...
+                .put("light", channel.shouldShowLights())
```
Comment (chemerisuk) [inline src/android/FirebaseMessagingPlugin.java:226]:
```suggestion
                .put("showLights", channel.shouldShowLights())
```

**Review by chemerisuk — CHANGES_REQUESTED** [review]
@WouterWidgets thanks! I'd agree to merge your PR after couple of minor changes.

File: src/android/FirebaseMessagingPlugin.java (hunk 1 above)
Comment (WouterWidgets) [reply src/android/FirebaseMessagingPlugin.java:220]: Good suggestion. I've changed it and will push an update shortly.

File: src/android/FirebaseMessagingPlugin.java (hunk 2 above)
Comment (WouterWidgets) [reply src/android/FirebaseMessagingPlugin.java:225]: The name `badge` was derived from the part where you set the channel properties:

`src/android/FirebaseMessagingPlugin.java:177`
```java
channel.setShowBadge(options.optBoolean("badge", true));
```

Shouldn't these be named the same?

File: src/android/FirebaseMessagingPlugin.java (hunk 3 above)
Comment (WouterWidgets) [reply src/android/FirebaseMessagingPlugin.java:226]: Same situation; the name `light` was derived from the part where you set the channel properties:

`src/android/FirebaseMessagingPlugin.java:179`
```java
channel.enableLights(options.optBoolean("light", false));
```

Shouldn't the names when *getting* the values be the same as when  *setting* the values?

**Comment by chemerisuk** [comment]
@WouterWidgets good work, thanks!

**Comment by WouterWidgets** [comment]
Many thanks for merging! Are you creating a new release tag for this?
Cheers, Wouter

**Comment by chemerisuk** [comment]
@WouterWidgets sorry for delay, published in v6.1.1

**Comment by WouterWidgets** [comment]
Many thanks indeed!
Regards, Wouter

**Pull request merged**